### PR TITLE
[FD-372] Auth pages - mobile view

### PR DESCRIPTION
**Pull request by starkyru**

### CODE REVIEW

File: src/screens/Auth/Login/Login.tsx [hunk 1]
```diff
@@ -2,52 +2,60 @@ import * as React from 'react';
...
-export type ILoginOwnProps = StyledComponentProps;
+export interface ILoginOwnProps extends StyledComponentProps {
+  theme: Theme;
+}
 
 function LoginComponent({
   classes = {},
   login,
   status,
+  theme,
```
Comment (BigBadAlien) [inline src/screens/Auth/Login/Login.tsx]: Not used

File: src/screens/Auth/Login/Login.tsx [hunk 2]
```diff
@@ -2,52 +2,60 @@ import * as React from 'react';
...
+import withTheme from '@material-ui/core/styles/withTheme';
 
 export interface ILoginReduxProps {
   login: (email?: string, password?: string, rememberMe?: boolean) => void;
   status: RequestStatus;
 }
 
-export type ILoginOwnProps = StyledComponentProps;
+export interface ILoginOwnProps extends StyledComponentProps {
+  theme: Theme;
```
Comment (BigBadAlien) [inline src/screens/Auth/Login/Login.tsx]: Not used?

File: src/screens/Dashboard/panels/Transactions/Transactions.tsx [hunk 3]
```diff
@@ -41,6 +41,10 @@ function TransactionsComponent({
     }
   }, []);
 
+  if (!address) {
+    return null;
```
Comment (BigBadAlien) [inline src/screens/Dashboard/panels/Transactions/Transactions.tsx]: The exceptional situation for the component ;)

Comment (starkyru) [reply src/screens/Dashboard/panels/Transactions/Transactions.tsx]: This is a situation when no wallet is been generated yet. now it just showing a blank panel

**Pull request merged**